canonical/ubuntu.com
# WD-10438 copy updates

## Done

According to [Jira task](https://warthogs.atlassian.net/browse/WD-10438):

- [/download/iot/installation-media](https://ubuntu.com/download/iot/installation-media) should redirect to [/core/docs/install-on-a-device](https://ubuntu.com/core/docs/install-on-a-device)
- In [/download/intel-nuc-desktop](https://ubuntu.com/download/intel-nuc-desktop) point “installation media instructions” in step 2 to [/tutorials/install-ubuntu-desktop](https://ubuntu.com/tutorials/install-ubuntu-desktop)

## QA

- Go to https://ubuntu-com-13942.demos.haus/download/iot/installation-media should redirect to https://ubuntu-com-13942.demos.haus/core/docs/install-on-a-device
- Go to https://ubuntu-com-13942.demos.haus/download/intel-nuc-desktop, find the "Installation media instructions" link, and click on it, it should go to https://ubuntu-com-13942.demos.haus/tutorials/install-ubuntu-desktop


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10438

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
